cluster/src/test/scala/io/snappydata/benchmark/snappy/tpcds/QueryExecutionJob.scala

```diff
@@ -0,0 +1,147 @@
+package io.snappydata.benchmark.snappy.tpcds
+
+import java.io.{File, FileOutputStream, PrintStream}
+
+import org.apache.spark.Logging
+import com.typesafe.config.Config
+import io.snappydata.benchmark.TPCH_Queries
+import io.snappydata.benchmark.snappy.tpch.QueryExecutor
+
+import org.apache.spark.sql.catalyst.util.fileToString
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.{Row, SnappyJobInvalid, SnappyJobValid, SnappyJobValidation, SnappySQLJob, SnappySession}
+
+/**
+  * Created by kishor on 10/1/18.
```

Comment:
Author section needs not to be checked-in. Can you remove from all files, wherever it exists.